src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteClient.java

```diff
@@ -66,6 +70,47 @@ public static MapRouletteClient instance(final MapRouletteConfiguration configur
         return null;
     }
 
+    /**
+     * This method writes challenge ids to a file.
+     *
+     * @param challengeId
+     *            challenge id of the newly created MapRoulette challenge.
+     * @param projectId
+     *            project id of the MapRoulette challenge.
+     */
+    public static void writeChallengeIdsToFile(final long challengeId, final long projectId)
+    {
+        outputPath.ifPresent(fileName ->
+        {
+            try
+            {
+                final BufferedWriter fileWriter = new BufferedWriter(
+                        new FileWriter(fileName, true));
+                fileWriter
+                        .append(String.format("project: %d;challenge: %d", projectId, challengeId));
```

Comment:
To making parsing this file easier, do you mind removing the whitespace between `project:` and the id

Reply:
Done